api.raml

```diff
@@ -193,6 +186,30 @@ schemas:
             "required": false,
             "type": "string",
             "enum": ["yes", "no"]
+          },
+          "password": {
```

Comment:
It may help us if we split out and/or clarify the purpose of some of the schemas in our RAML file. For example, `User` is being used both for what to supply when creating a User and for what comes back when getting a User... which I think won't be the same. (For example, AFAIK we won't want to let a POST provide a value for when the password expires... that's all calculated at the time the password is updated, and stored in the database then). At a subtler level, whether certain fields are required differs between those situations as well.

I'm not sure I actually address your question about suppressing the password info on an update... if not, could you help me understand what you mean?